Mathlib/GroupTheory/FreeGroup/Basic.lean

```diff
@@ -378,10 +378,37 @@ theorem eqvGen_step_iff_join_red : EqvGen Red.Step L₁ L₂ ↔ Join Red L₁ L
 
 end FreeGroup
 
-/-- The free group over a type, i.e. the words formed by the elements of the type and their formal
-inverses, quotient by one step reduction. -/
-@[to_additive "The free additive group over a type, i.e. the words formed by the elements of the
-  type and their formal inverses, quotient by one step reduction."]
+/--
+If `α` is a type, then `FreeGroup α` is the free group generated by `α`.
+This is a group equipped with a function `FreeGroup.of : α → FreeGroup α` which has
+the following universal property: if `G` is any group, and `f : α → G` is any function,
+then this function is the composite of `FreeGroup.of` and a unique group homomorphism
+`FreeGroup.lift f : FreeGroup α →* G`.
+
+A typical element of `FreeGroup α` is a formal product of
+elements of `α` and their formal inverses.
+For example if `x` and `y` are terms of type `α` then `x⁻¹ * y * y * x / y` is a
+"typical" element of `FreeGroup α`. In particular if `α` is empty
```

Comment:
The "typical element" description misses the key part that there's a quotient under reduction, which was present in the older docstring. Let's preserve that bit!